src/main.py

```diff
@@ -88,7 +88,7 @@ def hide(self, appname):
 class StopApp:
     def create(self):
         with open("/home/phablet/.local/share/applications/stop-waydroid.desktop", "w") as f:
-            f.write("[Desktop Entry]\nType=Application\nName=Waydroid Stop\nExec=waydroid session stop\nIcon=/usr/lib/waydroid/data/AppIcon.png")
+            f.write("[Desktop Entry]\nType=Application\nName=Waydroid Stop\nExec=waydroid session stop\nIcon=/usr/share/icons/hicolor/512x512/apps/waydroid.png")
```

Comment:
I was gonna say that's the XDG standard path and we can probably just do `Icon=waydroid` now, except I remembered https://gitlab.com/ubports/development/core/packaging/waydroid/-/blob/ubports/focal/debian/patches/0002-.desktop-Various-tweaks-fixes-for-Ubuntu-Touch.patch probably still has a reason to do the same there in `makeWaydroidDesktopFile()`

Reply:
Probably needs a fix in Lomiri: https://gitlab.com/ubports/development/core/packaging/waydroid/-/blob/ubports/focal/debian/patches/0002-.desktop-Various-tweaks-fixes-for-Ubuntu-Touch.patch#L10-11